pkg/report/linux.go

```diff
@@ -695,7 +705,7 @@ var linuxOopses = []*oops{
 				corrupted: true,
 			},
 			{
-				title: compile("INFO: task .* blocked for more than [0-9]+ seconds(?:.*\\n){0,10}Call Trace:\\n(?:.*(?:sched|_lock|completion|kthread).*\\n)* (?:{{PC}} )?{{FUNC}}"),
+				title: compile("INFO: task .* blocked for more than [0-9]+ seconds(?:.*\\n){0,10}Call Trace:\\n" + stacktraceRe([]string{"sched", "_lock", "completion", "kthread"})),
```

Comment:
We need to somehow make this work for guilty file as well.

Reply:
Added a few tests, had to add `kernel/sched/core.c` to guilty file blacklist, other than that everything seems to be working as expected.

Reply:
Actually found another file to be blacklisted, will update soon.

Reply:
Done. Ended up blacklisting `kernel/sched/*.c`, `kernel/time/timer.c` and `kernel/kthread.c` (don't have a good test for the last one yet).